book/src/SUMMARY.md

```diff
@@ -16,6 +16,8 @@
       - [CSINode Object](csi-node-object.md)
     - [Features](features.md)
         - [Secrets & Credentials](secrets-and-credentials.md)
+            - [StorageClass Secrets](secrets-and-credentials-external-provisioner.md)
```

Comment:
should the page names also be suffixed with Storage/SnapshotClass instead of sidecar?